### PR TITLE
Wlog console output to stderr

### PR DESCRIPTION
**Pull request by akallabeth**
Fixes the issue reported in #2595 

* Using ```stderr``` for all warning / error type messages
* Using ```stdout``` for info and debug type messages
* Skipping message printing if ```WLOG_OFF``` is set now

### CODE REVIEW

File: winpr/libwinpr/utils/wlog/ConsoleAppender.c [hunk 1]
```diff
@@ -211,11 +233,11 @@ wLogConsoleAppender* WLog_ConsoleAppender_New(wLog* log)
 	ConsoleAppender->WritePacketMessage =
 			(WLOG_APPENDER_WRITE_PACKET_MESSAGE_FN) WLog_ConsoleAppender_WritePacketMessage;
 
-	ConsoleAppender->outputStream = WLOG_CONSOLE_STDOUT;
+	ConsoleAppender->outputStream = WLOG_CONSOLE_DEFAULT;
 
 #ifdef _WIN32
-    if (IsDebuggerPresent())
-        ConsoleAppender->outputStream = WLOG_CONSOLE_DEBUG;
+		if (IsDebuggerPresent())
```
Comment (mfleisz) [inline winpr/libwinpr/utils/wlog/ConsoleAppender.c]: Formatting looks screwed

**Comment by mfleisz** [comment]
+1

**Comment by hardening** [comment]
@akallabeth looks like this kind of behaviour (splitting between stdout and stderr) is quite unsual for log systems. Do you have a use case in mind ?

**Comment by akallabeth** [comment]
@hardening No special case, just like it when I can redirect error messages easily.

**Comment by hardening** [comment]
+1

**Comment by bmiklautz** [comment]
+1

**Pull request merged**